routers/web/user/setting/applications.go

```diff
@@ -44,7 +45,7 @@ func ApplicationsPost(ctx *context.Context) {
 	_ = ctx.Req.ParseForm()
 	var scopeNames []string
 	for k, v := range ctx.Req.Form {
-		if strings.HasPrefix(k, "scope-") {
+		if strings.HasPrefix(k, issues_model.ScopeSortPrefix) {
```

Comment:
Oops, this change is not right.

This scope is `access token scope`, but not `label scope`